tox.ini

```diff
@@ -43,6 +43,8 @@ passenv =
   ROOKCHECK_*
   VERIFY_SSL_CERT
   XDG_CACHE_HOME
+  SSH_USE_EXISTING_AGENT
+  SSH_AUTH_SOCK
```

Comment:
I guess this is the only needed line if it's only for libvirt.

Reply:
SSH_AUTH_SOCK is not set by default because tox resets variables.

However we need the extra code because otherwise rookcheck is creating its own agent with a different socket.

Reply:
`SSH_AUTH_SOCK` needs to be passed in to tox (via adding it to `tox.ini`). With that, the socket is available and this should just work for libvirt.
And `rookcheck` should still create it's own `ssh-agent` (including a different `SSH_AUTH_SOCK`) that is used for interacting with the nodes over ssh.
I don't see why we need this code (beside the single line)

Reply:
It works only with SSH_AUTH_SOCK in tox indeed.

Then the log is misleading because when I read
`SSH_AUTH_SOCK=/tmp/rookcheck/root-rookci-29afb84a/ssh-agent.sock; export SSH_AUTH_SOCK;`
which is the first log message. I thought this was necessary.